src/lib/device/virtual_dev.hpp

```diff
@@ -299,6 +299,7 @@ class VirtualDev {
 
 private:
     uint64_t to_dev_offset(BlkId const& b, Chunk** chunk) const;
+    bool is_chunk_loaded(cshared< Chunk >& chunk) const;
```

Comment:
maybe is_chunk_exists or is_chunk_available is a better naming?  loaded sounds like if not you can load again:)